sources/install.sh

```diff
@@ -2994,6 +3021,372 @@ function install_cowpatty() {
+  # add-test-command "xfreerdp /version"
+  add-test-command "which xfreerdp"
+}
+
+function install_patator() {
+  colorecho "Installing patator"
+  fapt patator # messes up with lib, it installs python3-impacket (0.9.22-2)
+}
+
+function install_exiftool() {
+  colorecho "Installing exiftool"
+  fapt exiftool
+  add-test-command "wget -O /tmp/duck.png https://play-lh.googleusercontent.com/A6y8kFPu6iiFg7RSkGxyNspjOBmeaD3oAOip5dqQvXASnZp-Vg65jigJJLHr5mOEOryx && exiftool /tmp/duck.png && rm /tmp/duck.png"
+}
+
+function install_exifprobe() {
+  colorecho "Installing exifprobe"
+  fapt exifprobe
+  add-test-command "exifprobe -V; exifprobe -V |& grep 'Hubert Figuiere'"
+}
+
+function install_dnsenum() {
+  colorecho "Installing dnsenum"
+  fapt dnsenum
+  add-test-command "dnsenum --help; dnsenum --help |& grep 'Print this help message'"
+}
+
+function install_hydra() {
+  colorecho "Installing hydra"
+  fapt hydra
+  add-test-command "hydra --help; hydra -help |& grep 'more command line options'"
+}
+
+function install_imagemagick() {
+  colorecho "Installing imagemagick"
+  fapt imagemagick
+  add-test-command "convert -version"
+}
+
+function install_ascii() {
+  colorecho "Installing ascii"
+  fapt ascii
+  add-test-command "ascii -v"
+}
+
+function install_avrdude() {
+  colorecho "Installing avrdude"
+  fapt avrdude
+  add-test-command "avrdude '-?'"
+}
+
+function install_minicom() {
+  colorecho "Installing minicom"
+  fapt minicom
+  add-test-command "minicom --version; minicom --version |& grep 'This program is free software'"
+}
+
+function install_nasm() {
+  colorecho "Installing nasm"
+  fapt nasm
+  add-test-command "nasm --version"
+}
+
+function install_wabt() {
+  colorecho "Installing wabt"
+  fapt wabt
+}
+
+function install_ltrace() {
+  colorecho "Installing ltrace"
+  fapt-noexit ltrace
+}
+
+function install_strace() {
+  colorecho "Installing strace"
+  fapt strace
+}
+
+function install_stegosuite() {
+  colorecho "Installing stegosuite"
+  fapt stegosuite
+  add-test-command "stegosuite --help"
+}
+
+function install_steghide() {
+  colorecho "Installing steghide"
+  fapt steghide
+  add-test-command "steghide --version"
+}
+
+function install_binwalk() {
+  colorecho "Installing binwalk"
+  fapt binwalk
+  add-test-command "binwalk --help"
+}
+
+function install_foremost() {
+  colorecho "Installing foremost"
+  fapt foremost
+  add-test-command "foremost -V"
+}
+
+function install_pst-utils() {
+  colorecho "Installing pst-utils"
+  fapt pst-utils
+}
+
+function install_reaver() {
+  colorecho "Installing reaver"
+  fapt reaver
+  add-test-command "reaver --help; reaver --help |& grep 'Tactical Network Solutions'"
+}
+
+function install_bully() {
+  colorecho "Installing bully"
+  fapt bully
+  add-test-command "bully --version"
+}
+
+function install_cowpatty() {
+  colorecho "Installing cowpatty"
+  fapt cowpatty
+  add-test-command "cowpatty -V"
+}
+
```

Comment:
All this is duplicate code

Reply:
It's due to an error with a merge between 2 branch :/